nservicebus/operations/metrics.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Metrics
+summary: Measuring the performance and health of an endpoint.
+reviewed: 2017-03-24
+component: Metrics
+related:
+ - samples/metrics
+---
+
+Metrics collects information to measure endpoint health and performance.
+
+
+## Enabling Metrics
+
+snippet: Metrics-Enable
+
+
+## Reporting Metrics data
```

Comment:
Casing in the headings is inconsistent. E.g.

- Reporting Metrics data
- To NServiceBus Log
- To trace log
- Metrics Captured

Doesn't seem we're consistent in other docs (including the PerfCounters one) but my preference is to make everything lowercase where possible.